services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/resource_tracker.py

```diff
@@ -53,11 +45,12 @@ async def _startup() -> None:
     return _startup
 
 
-def on_app_shutdown(app: FastAPI) -> Callable[[], Awaitable[None]]:
+def on_app_shutdown(
+    app: FastAPI,  # pylint: disable=unused-argument
+) -> Callable[[], Awaitable[None]]:
     async def _stop() -> None:
-        if app.state.resource_tracker_rabbitmq_consumer:
-            await _unsubscribe_from_rabbitmq(app)
-            await app.state.rabbitmq_client.close()
+        # NOTE: We want to have persistent queue, therefore we will not unsubscribe
```

Comment:
Q? is this queue set as durable as well?

Reply:
Yes we have durable=True :+1: